app/styles/partials/buttons.scss

```diff
@@ -8,3 +8,23 @@
   box-shadow: 0 0 0 1px transparent inset;
   color: rgba(0, 0, 0, 0.95);
 }
+
+.ui.button.update-changes {
+  margin-bottom: 10px;
+}
+
+.ui.button.register {
```

Comment:
These are all specific to a component (register component to be specific. The register button isn't gonna be used anywhere else right). What is this doing in the file for buttons ?